components/script/animations.rs

```diff
@@ -35,6 +35,31 @@ impl Animations {
         }
     }
 
+    pub(crate) fn update_for_new_timeline_value(
+        &mut self,
+        window: &Window,
+        now: f64,
+    ) -> AnimationsUpdate {
+        let mut update = AnimationsUpdate::new(window.pipeline_id());
+        let mut sets = self.sets.write();
+
+        for set in sets.values_mut() {
+            // When necessary, iterate our running animations to the next iteration.
+            // TODO(mrobinson): We need to send animationiteration events here.
```

Comment:
This Todo can be removed, right?

Reply:
Yep. I've updated the branch, removing this commend.